From c61b42723fabe03dd858f3640dccf513b5b934ef Mon Sep 17 00:00:00 2001
From: Tim Gover <tim.gover@raspberrypi.org>
Date: Tue, 16 Oct 2018 11:41:30 +0100
Subject: [PATCH 215/220] ASoC: Fix compatible string for Allo Piano DAC

The RPi PCM512x had the wrong compatible for the Allo Piano DAC which it
replaced. Change this to the correct value.
---
 sound/soc/bcm/rpi-pcm512x-soundcard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/bcm/rpi-pcm512x-soundcard.c b/sound/soc/bcm/rpi-pcm512x-soundcard.c
index 6628920..7520142 100644
--- a/sound/soc/bcm/rpi-pcm512x-soundcard.c
+++ b/sound/soc/bcm/rpi-pcm512x-soundcard.c
@@ -255,7 +255,7 @@ static void snd_rpi_justboom_dac_shutdown(struct snd_pcm_substream *substream)
 };
 
 static const struct of_device_id snd_rpi_pcm512x_of_match[] = {
-	{ .compatible = "allo,allo-piano-dac",
+	{ .compatible = "allo,piano-dac",
 		.data = (void *) &drvdata_allo_piano_dac },
 	{ .compatible = "dionaudio,dionaudio-loco-v2",
 		.data = (void *) &drvdata_dionaudio_loco_v2 },
-- 
1.9.1

